pyaedt/modules/Material.py

```diff
@@ -2511,6 +2511,47 @@ def is_dielectric(self, threshold=100000):
         """
         return not self.is_conductor(threshold)
 
+    @pyaedt_function_handler
+    def set_djordjevic_sarkar_model(
+            self,
+            dk=4,
+            df=0.02,
+            i_freq=1e9,
+            sigma_dc=1e-12,
+            freq_hi=159.15494e9,
+    ):
+        """Set Djordjevic-Sarkar model.
+        Parameters
+        ----------
+        dk : int, float, str, optional
+            Dielectric constant at input frequency.
+        df : int, float, str, optional
+            Loss tangent at input frequency.
```

Comment:
```suggestion
            Loss tangent at input frequency. The default is ``0.02``.
```
I don't know why GitHub isn't letting me comment on a bunch of lines in a single comment!